composer.json

```
@@ -15,13 +15,21 @@
         "php-http/httplug": "^2.0",
         "react/http": "^1.0",
         "react/event-loop": "^1.0",
```

```suggestion
        "react/event-loop": "^1.2",
```

the deprecation and new method were introduced in 1.2 from what i can see. we should raise our minimum version (as it is just a minor version, i see no problem with that) or we would have to provide BC for when Loop::get does not exist.